# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1289894470 21600
#      Tue Nov 16 02:01:10 2010 -0600
# Node ID ac2d29f06fcef58bf75348ac0c3def51e70a2330
# Parent  d682cd6669acb322e11aa02d90945f33d6a8f982
unimplemented.m: Update unimplemented functions list.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-11-16  Ben Abbott <bpabbott@mac.com>
+
+	* miscellaneous/unimplemented.m: Update unimplemented functions list.
+
 2010-11-15  Rik  <octave@nomad.inbox5.com>
 
 	* plot/saveas.m: Add function name to error messages.  Use semicolons
 	to prevent unnecessary output.  Use common terms 'ext' for extension
 	and 'filename' for filename in docstring.
 
 2010-11-15  Kai Habel  <kai.habel@gmx.de>
 
diff --git a/scripts/miscellaneous/unimplemented.m b/scripts/miscellaneous/unimplemented.m
--- a/scripts/miscellaneous/unimplemented.m
+++ b/scripts/miscellaneous/unimplemented.m
@@ -184,17 +184,16 @@ function list = missing_functions ()
   "flow", 
   "fminsearch", 
   "frame2im", 
   "freqspace", 
   "funm", 
   "gallery", 
   "gammaincinv", 
   "gco", 
-  "getappdata", 
   "getframe", 
   "getpixelposition", 
   "getpref", 
   "gmres", 
   "grabcode", 
   "graymon", 
   "gsvd", 
   "guidata", 
@@ -228,17 +227,16 @@ function list = missing_functions ()
   "importdata", 
   "inmem", 
   "inputParser", 
   "inputdlg", 
   "inspect", 
   "instrfind", 
   "instrfindall", 
   "interpstreamspeed", 
-  "isappdata", 
   "iscom", 
   "isinterface", 
   "isjava", 
   "isocaps", 
   "ispref", 
   "isprop", 
   "isstudent", 
   "javaArray", 
@@ -338,27 +336,24 @@ function list = missing_functions ()
   "rbbox", 
   "rectangle", 
   "recycle", 
   "reducepatch", 
   "reducevolume", 
   "resample", 
   "reset", 
   "rgbplot", 
-  "rmappdata", 
   "rmpref", 
   "root", 
   "rotate", 
   "rotate3d", 
   "rsf2csf", 
-  "saveas", 
   "selectmoveresize", 
   "sendmail", 
   "serial", 
-  "setappdata", 
   "setpixelposition", 
   "setpref", 
   "showplottool", 
   "shrinkfaces", 
   "smooth3", 
   "snapnow", 
   "sound", 
   "soundsc", 
@@ -390,17 +385,16 @@ function list = missing_functions ()
   "tstool", 
   "uibuttongroup", 
   "uicontextmenu", 
   "uicontrol", 
   "uigetdir", 
   "uigetfile", 
   "uigetpref", 
   "uiimport", 
-  "uimenu", 
   "uiopen", 
   "uipanel", 
   "uipushtool", 
   "uiputfile", 
   "uiresume", 
   "uisave", 
   "uisetcolor", 
   "uisetfont", 
